webpack/scenes/ContentViews/Details/Filters/__tests__/contentViewFilters.test.js

```diff
@@ -90,3 +90,80 @@ test('Can search for filter', async (done) => {
   assertNockRequest(initialScope);
   assertNockRequest(searchResultScope, done);
 });
+
+test('Can remove a filter', async (done) => {
+  const { id } = firstFilter;
+  const autocompleteScope = mockAutocomplete(nockInstance, autocompleteUrl);
+
+  const getContentViewScope = nockInstance
+    .get(cvFilters)
+    .query(true)
+    .reply(200, cvFilterFixtures);
+
+  const removeFilterScope = nockInstance
+    .delete(api.getApiUrl(`/content_view_filters/${id}`))
+    .query(true)
+    .reply(200, {});
+
+  const callbackGetContentViewScope = nockInstance
+    .get(cvFilters)
+    .query(true)
+    .reply(200, {});
+
+  const { getAllByLabelText, getByText } = renderWithRedux(
+    <ContentViewFilters cvId={1} />,
+    renderOptions,
+  );
+
+  await patientlyWaitFor(() => {
+    expect(getAllByLabelText('Actions')[0]).toHaveAttribute('aria-expanded', 'false');
+  });
+  fireEvent.click(getAllByLabelText('Actions')[0]);
+  expect(getAllByLabelText('Actions')[0]).toHaveAttribute('aria-expanded', 'true');
+  await patientlyWaitFor(() => expect(getByText('Remove')).toBeTruthy());
```

Comment:
Any reason not to use `toBeInTheDocument`?
```suggestion
  await patientlyWaitFor(() => expect(getByText('Remove')).toBeInTheDocument());
```

Reply:
Is there a preference? I think I copied this from code seen in another test.

Reply:
It's not a strong preference but I feel like `toBeInTheDocument` evokes that we're testing for an HTML element, vs `toBeTruthy` which just evokes logic. But it's clear that we use both so I'm fine merging either way.

Reply:
If I change this here, I should likely change it in all the places to be consistent no?
Let me know if that's what you'd like to me do, I have no preference for either method, but prefer the method to be consistent  for future "pattent" copiers like myself. Lol.

Reply:
Sure, if it's consistency you want let's use `toBeInTheDocument` :)

Reply:
Done, all places where we are querying the document for a node, I have replaced toBeTruthy with toBeInTheDocument